Source/RapyutaSimulationPlugins/Private/Core/RRNetworkPlayerController.cpp

```diff
@@ -88,9 +88,10 @@ void ARRNetworkPlayerController::InitSimStateClientROS2()
     // Init SimStateClient's [ROS2Node] & [ClockPublisher]
     UWorld* currentWorld = GetWorld();
     SimStateClientROS2Node = currentWorld->SpawnActor<AROS2Node>();
+    SimStateClientROS2Node->AttachToActor(this, FAttachmentTransformRules::KeepRelativeTransform);
```

Comment:
If we need to attach the SimStateClient Ros2node to something, shouldn't we be attaching it to the SimulationStateClient instead of the Player controller?